librz/util/file.c

```diff
@@ -1257,3 +1257,40 @@ RZ_API char *rz_file_path_join(const char *s1, const char *s2) {
 	const char *sep = ends_with_dir ? "" : RZ_SYS_DIR;
 	return rz_str_newf("%s%s%s", s1, sep, s2);
 }
+
+/**
+ * \brief zip the contents of src and store in dst
+ * \param src source file (string containing filename)
+ * \param dst destination file (string containing filename)
+ * \return true, if successful; false otherwise
+ */
+RZ_API bool rz_file_zip(const char *src, const char *dst) {
```

Comment:
It's not ZIP. ZIP is a different format. I suggest to keep it as `rz_file_deflate()` and `rz_file_inflate()` instead.